dot/sync/syncer.go

```diff
@@ -46,7 +46,6 @@ type Service struct {
 	// Synchronisation variables
 	synced           bool
 	highestSeenBlock *big.Int // highest block number we have seen
-	runtime          runtime.Instance
```

Comment:
Also remove `Runtime` from `Config`.

Reply:
The test is using this runtime. I have created another issue to remove the runtime. #1674